packages/react/index.d.ts

```diff
@@ -1,57 +1,52 @@
-declare module "iframe-resizer-react" {
```

Comment:
Looking at this now. I notice this was wrong, it should have been
```js
  declare module "@iframe-resizer/react" {
```
Do you think that might have been what the issue was?

Reply:
You’re 100% right—I tried locally and that was the issue

Reply:
Thanks for checking